README.md

```diff
@@ -1 +1,112 @@
-# TDD
\ No newline at end of file
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Installation](#installation)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 TDD solver project <a name="about-project"></a>
+
+**TDD** project is a practice project to implement tests before methods, and implementing methods along the way.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack
+
+This project is built using the following technologies:
+
+<details>
+<summary>Programming Languages</summary>
+  <ul>
+    <li>Ruby</li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+<details>
+  <ul>
+    <li>Every method has tests</li>
+    <li>The tests were made before method implemented</li>
+  </ul>
+</details>
+
+
+<!-- GETTING STARTED -->
+
+## Getting Started <a name="getting-started"></a>
+
+To get started with the tdd project, follow the steps below:
+
+### Prerequisites
+
+Before running this project, make sure you have the following prerequisites installed:
+
+- Ruby
+- Rspec
+
+### Installation <a name="installation"></a>
+
+1. Clone this repository to your desired folder:
+
+   ```sh
+   git clone https://github.com/Ochiengsteven/TDD.git
+
+2. Navigate to the project directory:
+
+    ```sh
+    cd tdd
+3. Run the program:
+    ```sh
+    bundle exec rspec
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+👤 Ochieng Steven
+
+- GitHub: [Ochieng](https://github.com/Ochiengsteven)
+  
+👤 Edwin Gichuhi
+
+- GitHub: [Edwin](https://github.com/egichuhis)
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+<details>
+<summary>In near future we are planning to use this repo to practice TDD</summary>
+</details>
+
+<!-- CONTRIBUTING -->
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+[issues page](https://github.com/agneta1991/tdd/issues).
+
+## ⭐️ Show your support <a name="support"></a>
+<br>
+If you find this project useful, please consider giving it a star!
+
+[![Star](https://img.shields.io/github/stars/agneta1991/tdd?style=social)](https://github.com/Ochiengsteven/TDD)
+
+<!-- ACKNOWLEDGEMENTS -->
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+We would like to express our gratitude to the contributors and anyone who uses or supports this project.
+
+<!-- LICENSE -->
+## 📝 License <a name="license"></a>
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] the link to the MIT license is not working as expected, please make sure it is working as shown in the readme template provided by Microverse.